test-utils/src/main/java/apoc/util/GoogleCloudStorageContainerExtension.java

```diff
@@ -11,6 +11,7 @@
 
 import static java.net.HttpURLConnection.HTTP_OK;
 
+@SuppressWarnings("unused") // used from extended
```

Comment:
I would prefer moving this on extended as it's used only in there as you did for many other APIs